src/Administration/Resources/app/administration/src/module/sw-sales-channel/component/sw-sales-channel-measurement/sw-sales-channel-measurement.html.twig

```diff
@@ -0,0 +1,36 @@
+<sw-container
+    columns="1fr 1fr 1fr"
+    gap="0 30px"
+>
+    <sw-entity-single-select
+        ref="measurementSystemSelect"
+        v-model:value="salesChannel.measurementSystemId"
+        class="sw-sales-channel-detail__select-measurement-system"
+        entity="measurement_system"
+        :label="$t('sw-sales-channel.detail.measurement.labelDefaultUnitSystem')"
+        :criteria="measurementSystemCriteria"
+        @update:value="onMeasurementSystemChange"
+    />
+
+    <sw-entity-single-select
+        v-model:value="salesChannel.lengthUnitId"
+        entity="measurement_display_unit"
+        :label="$t('sw-sales-channel.detail.measurement.labelDefaultDimensionUnit')"
```

Comment:
Missing "(shortName)" on the label

Reply:
I don't see the shortName on the label here on the design. Just in the selection field.
![image](https://github.com/user-attachments/assets/74969f10-f66a-4485-ad1d-5ab7d438e0f8)

Reply:
Should we use the same label format for the select's options? @vienthuong

Reply:
Personally, I prefer to use the same label format, as showing full information to users is beneficial.